seedingScript.js

```diff
@@ -1,6 +1,6 @@
 const songData = require('./songData.js');
 
-var seederboi = () => {
+var seederboi = async () => {
```

Comment:
Why const and var? use let or only use var. var and (const and let) have different rules for scope.  mixed syntax invites bugs